data/strings/strings.txt

```diff
@@ -23454,3 +23454,11 @@
     vi = Dấu chân
     zh-Hans = 追踪
     zh-Hant = 追踪
+
+  [splash_subtitle]
+    comment = OpenStreetMap text on splash screen
+    tags = android,ios
+    en = Powered by OpenStreetMap
```

Comment:
```
en = Map data from OpenStreetMap
ar = OpenStreetMap ﻦﻣ ﺔﻄﻳﺮﺨﻟﺍ ﺕﺎﻧﺎﻴﺑ
be = Дадзеныя карты з OpenStreetMap
bg = Картографски данни от OpenStreetMap
cs = Mapová data z OpenStreetMap
da = Kortdata fra OpenStreetMap
de = Kartendaten von OpenStreetMap
el = Δεδομένα χάρτη από το OpenStreetMap
es = Datos de mapas de OpenStreetMap
eu = Maparen datuak OpenStreetMap-en
fa = OpenStreetMap ﺯﺍ ﻪﺸﻘﻧ ﯼﺎﻫ ﻩﺩﺍﺩ
fi = OpenStreetMapin karttatiedot
fr = Données cartographiques d'OpenStreetMap
he = OpenStreetMap-מ הפמ ינותנ
hu = Térképadatok az OpenStreetMapből
id = Data peta dari OpenStreetMap
it = Dati della mappa da OpenStreetMap
ja = OpenStreetMapからの地図データ
ko = OpenStreetMap의 지도 데이터
nb = Kartdata fra OpenStreetMap
nl = Kaartgegevens van OpenStreetMap
pl = Dane mapy z OpenStreetMap
pt = Dados cartográficos do OpenStreetMap
pt-BR = Dados do mapa do OpenStreetMap
ro = Date cartografice din OpenStreetMap
ru = Картографические данные из OpenStreetMap
sk = Mapové údaje z OpenStreetMap
sv = Kartdata från OpenStreetMap
sw = Data ya ramani kutoka OpenStreetMap
th = ข้อมูลแผนที่จาก OpenStreetMap
tr = OpenStreetMap'ten harita verileri
uk = Картографічні дані з OpenStreetMap
vi = Dữ liệu bản đồ từ OpenStreetMap
zh-Hans = 地图数据来自OpenStreetMap
zh-Hant = 來自 OpenStreetMap 的地圖數據
```

Can you please show a screenshot of how the longest line looks like?

Reply:
thank you :)
English & longest
![Screenshot_20220726-211944_Organic Maps (Debug)](https://user-images.githubusercontent.com/26939824/181104838-7ccbf060-2c45-49ba-9d9f-66b38445f6cb.png)